app/reducers/views/team.js

```diff
@@ -17,11 +17,33 @@ function lastTeamId(state = '', action) {
 
 function lastChannelForTeam(state = {}, action) {
     switch (action.type) {
-    case ViewTypes.SET_LAST_CHANNEL_FOR_TEAM:
+    case ViewTypes.SET_LAST_CHANNEL_FOR_TEAM: {
```

Comment:
Is changing the type of this going to cause any problems when the app gets updated? Should we maybe change the field name too?

Reply:
I think we don't want to lose the last channels you've seen when we do the upgrade, that's why, I can change the field name but the code for the migration will need to stick I think

Reply:
Should I really change the field name?

Reply:
I don't think it's really an issue to lose the last channel on upgrade. I always figured that PR was more for "the app was just in the background" as opposed to "we closed it on purpose". 0/5 though on whether or not we change it.